xero_python/accounting/models/tax_rate.py

```diff
@@ -310,6 +316,7 @@ def report_tax_type(self, report_tax_type):
             "ZRINPUT",
             "BADDEBT",
             "OTHERINPUT",
+            "None",
```

Comment:
Hi @SidneyAllen - we (@collabora) experienced an issue today that appeared related to validation of the `TaxType` field in some Xero API requests, and in particular a difference between the string values [`"None"`](https://github.com/XeroAPI/xero-python/pull/10/files#diff-f6a4407c12df003cc54b1fb117eedfa93786802bdf47371d29ae153958017ba9R319) and [`"NONE"`](https://github.com/XeroAPI/xero-python/pull/10/files#diff-f6a4407c12df003cc54b1fb117eedfa93786802bdf47371d29ae153958017ba9R260) as listed in this file.

Although this would create a bit of work on your side -- it's not urgent here, we have resolved the immediate concern -- it raised a few questions:

- Could the Xero API emit a less generic error message when one of these fields fails to validate server-side?
- Should both of the string values be in this list?
- If `None` should _not_ be present in this list, should it also be removed from some of the other lists (for example, [statuses](https://github.com/XeroAPI/xero-python/pull/10/files#diff-849d0b86d314a1b7ce4501f188974369e17e6000c7cf6d59e47e25d1b87b8031R137))?

(NB: we're not using `xero-python` currently)